doc-assets/node/18005-search-videos/18005.html

```diff
@@ -494,92 +414,51 @@ <h3>Open ended ranges</h3>
+        <td>starts_at is from the beginning of time to this moment</td>
+      </tr>
+      <tr>
+        <td>?q=schedule.starts_at:NOW</td>
+        <td>?q=schedule.starts_at:NOW</td>
+        <td>starts_at is from this moment onward</td>
+      </tr>
+      <tr>
+        <td>?q=schedule.ends_at:NOW..</td>
+        <td>?q=schedule.ends_at:NOW..</td>
+        <td>ends_at is from this moment to the end of time</td>
+      </tr>
+      <tr>
+        <td>?q=+schedule.starts_at:..NOW +schedule.ends_at:NOW..</td>
+        <td>?q=%2Bschedule.starts_at:..NOW%20%2Bschedule.ends_at:NOW..</td>
+        <td>starts_at before this moment and ends_at after this moment (video is in schedule this moment)</td>
+      </tr>
       </tbody>
     </table>
   </section>
 
-  <section class="bcls-section">
-    <h2 id="searchByState">Search by state</h2>
-
-    <p>You can perform or filter searches by the state of the video using the parameter:</p>
-
-<pre class="line-numbers"><code class="language-http">q=state:ACTIVE( | INACTIVE | PENDING | DELETED)<sup><strong><a href="#note3">[3]</a></strong></sup></code></pre>
-
-    <h3>Notes</h3>
-
-    <ul>
-      <li id="note3"><sup><strong>[3]</strong></sup> Search for DELETED videos is only available for videos deleted in the past 10 days (the current time minus 10 days), and only through the CMS API (not the Playback API).</li>
-    </ul>
-  </section>
-
-  <section class="bcls-section">
-    <h2 id="clips">Clip search terms</h2>
-
-    <p>Clips are videos created from sections of other videos. Clips can be generated by <a href="/node/16672">Brightcove Social</a>, and other means will be available in the future. There are some special search terms you can use to find generated clips in an account:</p>
-
-    <ul>
-      <li><code>q=%2Bis_clip:true</code> - returns only clips</li>
-      <li><code>q=-is_clip:true</code> - returns only non-clips</li>
-      <li><code>q=%2Bis_clip:false</code> - returns only non-clips (functionally identical to the previous item)</li>
-      <li><code>q=%2Bclip_source_video_id: <span class="bcls-input">video_id</span></code> - returns clips generated from the specified video</li>
-    </ul>
-  </section>
-
+  <!-- Combined Search -->
   <section class="bcls-section">
     <h2 id="combine_search_criteria">Combine search criteria</h2>
 
     <p>You can combine criteria for complex searches.</p>
 
     <p>Example: This request searches for videos with a <code>name</code> value of <strong>gossip</strong>, which were updated between August 1, 2010 and October 8, 2010. It then sorts the response data by <code>updated_at</code> date in descending order.</p>
 
-<pre class="line-numbers"><code class="language-http">q=%2Bname:gossip+%2Bupdated_at:2010-08-01T00:00:00Z..2010-10-08T00:00:00Z&amp;sort=-updated_at</code></pre>
+    <pre class="line-numbers"><code class="language-http">q=%2Bname:gossip%20%2Bupdated_at:2010-08-01..2010-10-08&amp;sort=-updated_at</code></pre>
```

Comment:
We have plenty of examples with timestamps.  Once again the description was wrong, so I simply removed the time component.

Reply:
How does this behave though? The lack of times seems like this could be far less intuitive. If I specified the query `updated_at:2010-08-01..2010-08-02` then it may be natural to assume that this would include results for any videos which had been updated during those two days, but if the times are zeroed out then the results would effectively only include results from `2010-08-01` (or the first instant of `2010-08-02`). Supporting a decent experience around this is a bigger question but this change may aggravate confusion if that's the behavior, or the docs should be spelled out to indicate the _mostly_ exclusive end date.
With the more powerful date range syntax the exact behavior could be specified more intuitively using something like:
`[2010-08-01 TO 2010-08-03}` where the `}` indicates an exclusive end and therefore exactly what would be desired if searching for the results from those two dates.